app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -178,4 +210,20 @@ def process_scan_images(images)
       end
     end
   end
+
+  def process_check_images_compliance(images)
+    ContainerImage.where(:id => images).order("lower(name)").each do |image|
+      image_name = image.name
```

Comment:
Just a minor thing, but, does `image.name` really need to be copied to a local variable?